OpenAI_API/Completions/CompletionEndpoint.cs

```diff
@@ -110,7 +111,9 @@ public Task<CompletionResult> CreateCompletionsAsync(CompletionRequest request,
 			double? presencePenalty = null,
 			double? frequencyPenalty = null,
 			int? logProbs = null,
-			params string[] stopSequences)
+			bool? echo = false,
```

Comment:
since `echo` is a Nullable, this should default to `null` to indicate unspecified (in which case line 128 below will use the default value).  With is set to `false` here, it will not fall back to the default value in line 128 (`Echo = echo ?? DefaultCompletionRequestArgs.Echo`).

Reply:
understood. this makes sense. updating now.